docs/products/postgresql/reference/log-formats-supported.rst

```diff
@@ -0,0 +1,18 @@
+Log formats supported
+=====================
+
+Aiven for PostgreSQL® supports setting different log formats which are compatible with popular log analysis tools like ``pgbadger`` or ``pganalyze``.
+
+You can use this functionality by navigating to your PostgreSQL® service on the `Aiven console <https://console.aiven.io/>`_.  
+
+Scroll down to the end of the overview page under ``Advanced configuration`` and select the ``Change`` button.  You can then select the ``pg.log_line_prefix`` parameter where there is a set of formats available that can be viewed from the drop down menu:  
+
+* ``'pid=%p,user=%u,db=%d,app=%a,client=%h '``
+* ``'%t [%p]: [%l-1] user=%u,db=%d,app=%a,client=%h '``
+* ``'%m [%p] %q[user=%u,db=%d,app=%a] '``
```

Comment:
Are those formats self explanatory? or should we add an example of the output?

Reply:
Those are the drop-down selection of options that was just copied over:
![image](https://user-images.githubusercontent.com/10273586/211942067-cc23dd25-8545-4d33-93d9-b1d391bd2d3f.png)

I can't quite say they're self explanatory, but I think you may have a good point of possibly adding an example output.  I'll give that a test run and see if I can do so.  The original article didn't have one so I thought it was self explanatory for others.

Reply:
Ok I provided the outputs of each option.  They look very similar and only have some minor differences if you compare them side-by-side.  Pushing everything on the next commit.